File: langgraph/result/qwen/7b-eq2js/63.js
function calculateModifiedFibonacci(index) {
    if (index === 0 || index === 1) {
        return 0;
    } else if (index === 2) {
        return 1;
    } else {
        return calculateModifiedFibonacci(index - 1) + 
               calculateModifiedFibonacci(index - 2) + 
               calculateModifiedFibonacci(index - 3);
    }
}

File: langgraph/data/semantic_equiv/63.py
def calculate_modified_fibonacci(index: int):
    if index == 0:
        return 0
    elif index == 1:
        return 0
    elif index == 2:
        return 1
    else:
        return calculate_modified_fibonacci(index - 1) + calculate_modified_fibonacci(index - 2) + calculate_modified_fibonacci(index - 3)

# method name: fibfib -> calculate_modified_fibonacci
# parameter name: n -> index
# added elif for clarity in condition checks
# no other changes in logic or functionality